Modules/Core/Common/src/itkLightProcessObject.cxx

```diff
@@ -29,13 +29,6 @@ ::LightProcessObject()
   m_Progress = 0.0f;
 }
 
-/**
- * Destructor for the LightProcessObject class. We've got to
- * UnRegister() the use of any input classes.
- */
-LightProcessObject
-::~LightProcessObject() = default;
-
```

Comment:
There are several *.cxx files which have the default destructor removed from them. It is my understanding that moving this method to be implicitly inline in the header, will create an instance of the method in every compilation unit. Where as with the prior "opaque" declaration in the header the library instance would have to be used.